README.md

```diff
@@ -108,8 +122,43 @@ $ yarn install
 
 You have several rake tasks available for you:
 
-* `rake development_app`: Creates a development app inside `decidim_development` which you can use to run an application with the gems in your path.
-* `rake test_all`: Generates a test app for every engine and runs their tests.
-* `rake generate_all`: Generates all the tests apps but doesn't run the tests - this is useful is you want to run them manually afterwards.
+* `bundle exec rake development_app`: Creates a development app inside `decidim_development` which you can use to run an application with the gems in your path.
+* `bundle exec rake test_all`: Generates a test app for every engine and runs their tests.
+* `bundle exec rake generate_all`: Generates all the tests apps but doesn't run the tests - this is useful is you want to run them manually afterwards.
+* `cd <component>` and do `bundle exec rspec spec` to run those particular tests.
+
+
+### Browse Participatory Processes
+
+After you create a development app (`bundle exec rake development_app`):
+- `cd development_app`
+- `bundle exec rails s`
+- Go to 'http://localhost:3000'
```

Comment:
Optionally, you can log in as: user@decidim.org | decidim123456

Also, if you want to verify yourself agains the default authorization use a document number ended with "X"